src/singleRequestProxyFactory.ts

```
@@ -0,0 +1,38 @@
+import {
+  ERC20SingleRequestProxyCreated,
+  EthereumSingleRequestProxyCreated,
+} from "../generated/SingleRequestProxyFactory/SingleRequestProxyFactory";
+import { SingleRequestProxyDeployment } from "../generated/schema";
+
+export function handleEthereumSingleRequestProxyCreated(
+  event: EthereumSingleRequestProxyCreated,
+): void {
+  let deployment = new SingleRequestProxyDeployment(
```

Biome

[error] This let declares a variable that is only assigned once.

'deployment' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)